managing_providers/_topics/embedded_workflows.md

```diff
@@ -0,0 +1,339 @@
+
+### Embedded Workflows UI
+
+For the *Embedded Workflows* feature, the following pages are available in the UI, which you can use to access and work with workflows. To view these pages, go to **Automation** > **Embedded Workflows** and select the page.
+
+![Embedded Workflow UI](../images/embedworkflow_ui.png)
+
+- **Workflows**
+
+   This UI page includes a summary of the available workflows. You can use different views to review the workflows:
+
+   - Text view, which is the raw representation (JSON or YAML) of the Amazon State Languages (ASL) code.
+   - Graph view, which shows a visual representation of the workflow structure.
+
+- **Repositories**
+
+   This page lists, and provides a summary of, the added embedded workflow repositories. From this page, you can add, update, and remove repositories.
+   When a repository is added and synchronized, any workflows that are found in the repository can be viewed on the **Workflows** UI page.
+
+- **Credentials**
+
+   Credentials are used by {{ site.data.product.title_short }} for any authentication that is required when running embedded workflows, and when importing workflows from a version control system.
+
+   This page lists, and provides a summary of, the credentials for accessing and using embedded workflows. From this page, you can add, update, and delete credentials.
+
+In addition to the *Embedded Workflows* UI, **Embedded Workflows** can also be used in Service Dialogs for dynamic dialog fields and with Service Catalog Items for provisioning, reconfiguration, and retirement entrypoints.
+
+* Service Dialog
+
+   When creating a service dialog, fields that are dynamic include the option for using **Embedded Workflows** or **Embedded Automate**.
+
+   ![Service Dialog options](../images/embedworkflow_servicedialogdynamic.png)
+
+* Service Catalog Items
+
+   The provisioning, reconfigure, and retirement entry points include the option for using **Embedded Workflows** or **Embedded Automate**.  You can set the point field to the workflow that you want to use for any, or all, of the three service entry points.
+
+   ![Entry Points options](../images/embedworkflow_entrypointsembedded.png)
+
+### Adding an Embedded Workflows Repository
+
+As a starting point to begin authoring a workflow, you must first add the repository that includes your workflows.
+
+To add a repository, complete the following steps:
+
+1. Click  **Automation** > **Embedded Workflows** > **Repositories** > **Configuration** > **Add New Repository**.
+
+   ![Add a repository](../images/embedworkflow_addnewrepository.png)
+
+2. Enter the repository name in the **Name** field.
+
+3. Add a description for the repository in the **Description** field.
+
+4. Enter a Git repository URL for the repository in the **GitHub URL** field. The Git repository URL does not need to be a GitHub repository.
```

Comment:
The field is just "URL" not "GitHub URL" https://github.com/ManageIQ/manageiq-documentation/pull/1752/files#diff-e473dd8e1cafe1f15527bc795eea54496c4e09ff40416a770384f11edb7a65f5

```suggestion
4. Enter a Git repository URL for the repository in the **URL** field. The Git repository URL does not need to be a GitHub repository.
```

Reply:
This is updated as requested.